src/Storages/S3Queue/S3QueueFilesMetadata.cpp

```diff
@@ -18,6 +18,7 @@
 #include <Poco/JSON/Object.h>
 #include <Poco/JSON/Parser.h>
 
+#include <numeric>
```

Comment:
because of `std::iota`
I removed some std headers in `ZooKeeper.h` which probably provided transiently `<numeric>` to this file.
C++ is beautiful...